<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Contact</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name *</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="placeholderText">
      <string>Enter full name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="phoneLabel">
     <property name="text">
      <string>Phone *</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phoneLineEdit">
     <property name="placeholderText">
      <string>Enter phone number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="emailLabel">
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="placeholderText">
      <string>Enter email address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="addressLabel">
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="addressTextEdit">
     <property name="placeholderText">
      <string>Enter address</string>
     </property>
     <property name="maximumHeight">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="notesLabel">
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="notesTextEdit">
     <property name="placeholderText">
      <string>Additional notes (optional)</string>
     </property>
     <property name="maximumHeight">
      <number>60</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
